src/main/java/net/sf/jabref/importer/EntryFromPDFCreator.java

```diff
@@ -64,7 +64,7 @@ public boolean accept(File f) {
         PdfImporter pi = new PdfImporter(JabRefGUI.getMainFrame(), JabRefGUI.getMainFrame().getCurrentBasePanel(), JabRefGUI.getMainFrame().getCurrentBasePanel().getMainTable(), -1);
```

Comment:
I'm sorry, but I have a hard time understanding comments like this. ~10 EDT bugs, 3 NPEs, 6 forgotten translations and the comment relates to the readability of a white line? 

Again, I think people hesitate touching bad code in the long run if the reviews focus more on what could be done as well (just because there are lines on the screen that are not perfect) instead of what has been done. No offense though and none taken.

(I think the readability is OK and if anything one should maybe just skip many of the arguments and get it from JabRefGUI in PdfImporter instead, but I haven't checked how it is called in other places.)